<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferencesPatternPage</class>
 <widget class="QWidget" name="PreferencesPatternPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>485</width>
    <height>570</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>485</width>
    <height>570</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>485</width>
    <height>570</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true">Pattern</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="undo_GroupBox">
     <property name="minimumSize">
      <size>
       <width>465</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>465</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>Undo</string>
     </property>
     <widget class="QLabel" name="undoCount_Label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>70</width>
        <height>16</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>70</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>70</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Count step:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QSpinBox" name="undoCount_SpinBox">
      <property name="geometry">
       <rect>
        <x>86</x>
        <y>20</y>
        <width>39</width>
        <height>20</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>20</y>
        <width>64</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string> (0 - no limit)</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="WorkPiece_GroupBox">
     <property name="minimumSize">
      <size>
       <width>465</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>465</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>Workpiece</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QCheckBox" name="forbidFlipping_CheckBox">
        <property name="toolTip">
         <string>By default forbid flipping for all new created workpieces</string>
        </property>
        <property name="text">
         <string>Forbid flipping</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="hideMainPath_CheckBox">
        <property name="toolTip">
         <string>By default hide the main path if the seam allowance was enabled</string>
        </property>
        <property name="text">
         <string>Hide main path</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="labelFont_Label">
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Label font:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QFontComboBox" name="labelFont_ComboBox"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="seamAllowance_GroupBox">
     <property name="minimumSize">
      <size>
       <width>465</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>465</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>Seam allowance</string>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="defaultSeamAllowance_Label">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Default value:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="defaultSeamAllowance_DoubleSpinBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="maximum">
         <double>100.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="notches_GroupBox">
     <property name="minimumSize">
      <size>
       <width>465</width>
       <height>150</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>465</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>Notches</string>
     </property>
     <widget class="QCheckBox" name="showSecondNotch_CheckBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>343</width>
        <height>17</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Show notch on both the seam allowance and seam line.</string>
      </property>
      <property name="text">
       <string>Show second notch on seam line</string>
      </property>
     </widget>
     <widget class="QComboBox" name="defaultNotchType_ComboBox">
      <property name="geometry">
       <rect>
        <x>85</x>
        <y>45</y>
        <width>120</width>
        <height>20</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="defaultNotchLength_DoubleSpinBox">
      <property name="geometry">
       <rect>
        <x>85</x>
        <y>71</y>
        <width>120</width>
        <height>20</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="decimals">
       <number>3</number>
      </property>
      <property name="minimum">
       <double>0.125000000000000</double>
      </property>
      <property name="maximum">
       <double>1.500000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.010000000000000</double>
      </property>
      <property name="value">
       <double>0.125000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="defaultNotchType_Label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>47</y>
        <width>70</width>
        <height>13</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>70</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>Default type:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="defaultNotchLength_Label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>73</y>
        <width>70</width>
        <height>13</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>70</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Length:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="defaultNotchWidth_Label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>70</width>
        <height>13</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>70</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Width:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="defaultNotchWidth_DoubleSpinBox">
      <property name="geometry">
       <rect>
        <x>85</x>
        <y>98</y>
        <width>120</width>
        <height>20</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="decimals">
       <number>3</number>
      </property>
      <property name="singleStep">
       <double>0.050000000000000</double>
      </property>
      <property name="value">
       <double>0.250000000000000</double>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="Formats_GroupBox">
     <property name="minimumSize">
      <size>
       <width>465</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>465</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>Label data/time format</string>
     </property>
     <layout class="QFormLayout" name="formLayout_4">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::ExpandingFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="dateFormats_Label">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Date:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QComboBox" name="dateFormats_ComboBox">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>120</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="editDateFormats_PushButton">
          <property name="minimumSize">
           <size>
            <width>75</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Edit formats</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="timeFormats_Label">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Time:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QComboBox" name="timeFormats_ComboBox">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>120</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="editTimeFormats_PushButton">
          <property name="minimumSize">
           <size>
            <width>75</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Edit formats</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
